docs/SETUP.md

```diff
@@ -110,6 +110,22 @@ Finally, create the `ilpkit` database while logged in as the `postgres` user.
 
 ```sh
 postgres$ createdb ilpkit
+```
+
+For redundancy of data you may want to run this on a crontab. Outputing the file to
```

Comment:
Should the backup instructions go into a new section, separate from the setup DB instructions?